Integrations/integration-GRR.yml

```diff
@@ -34,8 +34,30 @@ script:
     var server = params.server + ':' + params.port;
     var insecure = params.insecure;
 
+    var getCSRFToken = function(url) {
+        var res = http(
+                url,
```

Comment:
Will it work with any URL?

Reply:
Any URL that is referenced in this integration, yes.  Function was created while referencing GRR Python API Client (https://github.com/google/grr/blob/master/api_client/python/grr_api_client/connectors/http_connector.py)

Reply:
I think its safer to use the server url rather than sending GET to each url, you might get a 405 error

Reply:
@Spivaka - That's speculative... The client source above shows if the CSRF variable is not present then it will perform GET request on specified URL endpoint.  I'm following GRR implementation on this one -- Ignoring it resulted in the generation of this PR